src/crystal/system/unix/mime.cr

```diff
@@ -0,0 +1,24 @@
+module Crystal::System::MIME
+  MIME_SOURCES = {
```

Comment:
Is there a way to only read the files that are on the OS?
maybe use `{% if flag(:os) %}` to include the files necessary.

Reply:
I don't know what the overhead is and anything involving IO seems like it could be impactful.  But I see your point and feel it could be investigated in a later PR.